### ISSUE TITLE
issue viewing multicameras from rviz or image_view

### ISSUE THREAD

**Issue by SISegwayRmp** [ros-drivers/pointgrey_camera_driver#23]
Here's my issue:

I launch 2 Flea3 USB3 cameras with separate namespaces and separate nodelet managers: works as expected

Everything works fine, the topics are publishing and I get this result, as expected:
![before_both_open_in_imageview](https://cloud.githubusercontent.com/assets/9217506/4632539/960b53a4-53be-11e4-8d57-05ef36fbc11d.png)

Now everything continues to work just fine if I open 1 image at a time in either image_view or rviz.

Here's the issue, when I open both images at the same time everything falls apart. I also tried launching image_view as a nodelet inside the individual namespaces with separate nodelet managers. I get a strange result after attempting to open simultaneously:
![after_both_open_in_imageview](https://cloud.githubusercontent.com/assets/9217506/4632582/edd0f666-53be-11e4-8e04-c9275beb6fed.png)

I can get both to display in image_view with a modified versions of the camera1394, libdc1394, and libusb. But not with the pointgrey_camera_driver.

I know this was brought up in a closed issue, but it did not have resolution there. Anyone have an idea why this might be happening? I am completely stumped....

**Comment by SISegwayRmp** [comment]
this was an issue related to this:

https://github.com/ros-drivers/pointgrey_camera_driver/issues/8

it can be closed as well

**Comment by kingfolk** [comment]
Hi Patrick,

I use your launch file https://github.com/ros-drivers/pointgrey_camera_driver/issues/8#issuecomment-70251545 for two flea3 cameras. But only the first camera is streaming, although i got topics from both left_camera and right_camera. I have already follow the instruction of usbfs setting. Do you have a clue of what might be wrong?

many thanks

**Comment by JakubWedrowycz** [comment]
Hello, 
I am trying to use two Pointgrey cameras, and also have this problem: 
[ERROR] [1446217164.145882434]: PointGreyCamera::grabImage Failed to retrieve buffer | FlyCapture2::ErrorType 41There is an image consistency issue with this image.

The cameras are connected, and I can see the topic for both, but once I want to see the stream from them, using image_view, the error occurs for image_raw topic. And to compare for image_mono, the window just opens but no stream is visible (white window) and no error message is sent. 
I tried increasing memory buffer, I followed the solutions from ROS Wiki but it doesn't help - still the same error.
The funny thing is, that my friend is using exactly the same launch file on his computer and the same version of driver and ROS, and it works for him strangely: he can see the stream but once in few seconds he receive the same error message, but the stream doesn't stop. 
Can anyone help with that? I would really appreciate it
